pkg/crc/config/settings.go

```diff
@@ -134,6 +135,8 @@ func RegisterSettings(cfg *Config) {
 
 	cfg.AddSetting(KubeAdminPassword, "", validateString, SuccessfullyApplied,
 		"User defined kubeadmin password")
+	cfg.AddSetting(DeveloperPassword, "", validateString, SuccessfullyApplied,
```

Comment:
What if we use `"developer"` here as the default value (second parameter to `AddSetting`), wouldn't this make the second commit redundant?

Reply:
This change was added in second commit: e9a11fc31e65e4e8d88beacac7effddba84ca152

Did you mean we can avoid third commit by adding developer as default value?